apps/api/src/domain/models/vocabulary-list/entities/vocabulary-list.entity.ts

```diff
@@ -268,6 +276,78 @@ export class VocabularyList extends Resource {
         return this;
     }
 
+    @UpdateMethod()
+    importEntries(importItems: VocabularyListEntryImportItem[]): ResultOrError<VocabularyList> {
+        if (this.entries.length > 0) {
+            return new FailedToImportEntriesToVocabularyListError(this.id, [
+                new InternalError(
+                    `you cannot import entries to a vocabulary list that already has entries`
+                ),
+            ]);
+        }
+
+        if (importItems.length === 0) {
+            return new FailedToImportEntriesToVocabularyListError(this.id, [
+                new InternalError(
+                    `you must provide at least one item when importing entries to a vocabulary list`
+                ),
+            ]);
+        }
+
+        const unknownFilterPropertyErrors = importItems
```

Comment:
We might want to update this to be able to say in the error message which properties where not registered.

Reply:
Done.